README.md

```diff
@@ -136,6 +137,31 @@ OktaAuth.login(username: "user@example.com", password: "password").start(view: s
 }
 ```
 
+## Logout
+
+First, update your `AppDelegate` to include the following function to allow the redirect to occur:
```

Comment:
We can skip this part, since updating `AppDelegate` is already required for sign-in.

Reply:
Done,